src/dot.rs

```diff
@@ -0,0 +1,51 @@
+use zed::settings::LspSettings;
+use zed_extension_api::{self as zed, serde_json, LanguageServerId, Result};
+
+struct DotLspExtension {}
+
+impl DotLspExtension {
+    fn language_server_binary_path(
+        &mut self,
+        _language_server_id: &LanguageServerId,
+        worktree: &zed::Worktree,
+    ) -> Result<String> {
+        return worktree
+            .which("dot-language-server")
+            .ok_or(String::from("Ensure dot-language-server is in PATH"));
```

Comment:
nitpick: clippy is being noisy about the unnecessary `return` here, can you remove please?

```suggestion
       worktree
            .which("dot-language-server")
            .ok_or(String::from("Ensure dot-language-server is in PATH"))
```